governance/reviews/template.md

```diff
@@ -19,35 +33,46 @@ Status: Exemplary|Satisfactory|Mostly Satisfactory|Needs Work
 
 ### Executing the Assessment
 
-<!--- A brief description that details the timebox the assessment occurred and the individuals involved in the assessment. --->
+<!--- A brief description that details the timebox the assessment occurred and the individuals involved in the assessment. 
+
+Make sure you use a snapshot of the governance documents for your assessment and note the commit hash of the snapshot here as a link.
+--->
 
 ### Must-Fix Items
 
 The following issues have been identified that need to be resolved before [project milestone or other requirement]:
-<!--- The items in the list should be summarized, have a prioritized ordering and are expected to be considered Critical or High, they are usually blockers to project advancement. For each item in this list, a corresponding detailed description should be placed in the Findings table. Note that which items are required depends on the project's maturity level. --->
+<!--- The items in the list should be summarized, have a prioritized ordering and are expected to be considered blockers to project advancement. For each item in this list, a corresponding detailed description should be placed in the Findings table. Note that which items are required depends on the project's maturity level.
+
+Items in the "Governance Findings Table" at the bottom with "Blocker" importance should be reported here as must-fix items.
+
+If there are no must-fix items, do not delete the section but write that there are no must-fix items.
```

Comment:
I think it is better to classify must-fix items as the ones that are listed as blocker at the end.

Existing severity levels were "Critical/High/Medium/Low" and it was said here that Critical and High are basically blockers. This was confusing for me.

I've made changes to the bottom of the document as well to make this grouping:
- Must-fix: Blocker
- Areas for improvement: High/Medium/Low

Reply:
I'm not sure that the distinction between "High" and "Blocker" would be clear.  Maybe omit "High" entirely, since all High will be Blockers anyway?

And ... why use "Blocker" instead of "MustFix"?

Reply:
> Maybe omit "High" entirely, since all High will be Blockers anyway?

My intention is along the same lines. I tried to make distinction between a blocker and a non-blocker.

Let me update with what you've suggested, which is even simpler.

So,
* 3 categories: critical/medium/low
* Critical --> blocker
* Medium/Low --> non-blocker

> And ... why use "Blocker" instead of "MustFix"?

I don't have a strong opinion in this one. I am ok with switching back to MustFix.

Reply:
Pushed a new commit for this change.

Reply:
Whatever term we use: "Critical", "Blocker", or "MustFix" should be the same in both the grid and the section header.